api/RTCPeerConnection.json

```diff
@@ -0,0 +1,1420 @@
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCPeerConnection/pendingLocalDescription",
+          "support": {
+            "webview_android": {
+              "version_added": "56"
+            },
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": "56"
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "22"
+            },
+            "firefox_android": {
+              "version_added": "44"
+            },
+            "ie": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": null
+            },
+            "opera_android": {
+              "version_added": null
+            },
+            "safari": {
+              "version_added": null
+            },
+            "safari_ios": {
+              "version_added": null
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "pendingRemoteDescription": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCPeerConnection/pendingRemoteDescription",
+          "support": {
+            "webview_android": {
+              "version_added": "56"
+            },
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": "56"
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "22"
+            },
+            "firefox_android": {
+              "version_added": "44"
+            },
+            "ie": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": null
+            },
+            "opera_android": {
+              "version_added": null
+            },
+            "safari": {
+              "version_added": null
+            },
+            "safari_ios": {
+              "version_added": null
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "remoteDescription": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCPeerConnection/remoteDescription",
+          "support": {
+            "webview_android": {
+              "version_added": "56"
+            },
+            "chrome": {
+              "version_added": true
```

Comment:
56.

This and several other methods became promise-based in Chrome 51/Opera 38. https://www.chromestatus.com/features/5743313768415232

Reply:
So like.. ?
```
"chrome": [
              {
                "version_added": "51",
                "notes": "Promised based version."
              },
              {
                "version_added": "56"
              }
            ],
```
Document reads like it was added before this too! :confused:

Reply:
First, go read the description in the link I provided. Some members actually because promise-based in Chrome 50. The description, written by the guy who did the actual work, explains which members were changed for each version.

Second, 56 is the version when the webkit prefix was removed. https://www.chromestatus.com/features/5312661013135360

Reply:
I've somehow still missed all the methods! I think last week I thought methods were excluded from this data. :man_facepalming:  Amazing what I've learned this last week. Won't be making these mistakes again in a hurry.